orbit/pkg/table/safari_options/safari_options.go

```diff
@@ -0,0 +1,53 @@
+//go:build darwin
+// +build darwin
+
+package safari_options
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	tbl_common "github.com/fleetdm/fleet/v4/orbit/pkg/table/common"
+	"github.com/osquery/osquery-go/plugin/table"
+	"strings"
+)
+
+// Columns is the schema of the table.
+func Columns() []table.ColumnDefinition {
+	return []table.ColumnDefinition{
+		table.TextColumn("user_name"), // required
+		table.TextColumn("show_full_url_in_smart_search_field"),
+	}
+}
+
+// Generate is called to return the results for the table at query time.
+// Constraints for generating can be retrieved from the queryContext.
+func Generate(ctx context.Context, queryContext table.QueryContext) ([]map[string]string, error) {
+	userName := ""
+	if userName = getUserNameFromConstraints(queryContext); userName == "" {
+		return nil, errors.New("missing user_name")
+	}
+
+	res, err := tbl_common.RunCommand(ctx, "/usr/bin/defaults", "read", "/Users/"+userName+"/Library/Containers/com.apple.Safari/Data/Library/Preferences/com.apple.Safari", "ShowFullURLInSmartSearchField")
```

Comment:
We don't need a new table for this (whenever you see `defaults read` we can use `plist`):
```
SELECT * FROM plist WHERE path LIKE '/Users/%/Library/Containers/com.apple.Safari/Data/Library/Preferences/com.apple.Safari.plist' AND key = 'ShowFullURLInSmartSearchField';
```

Reply:
Haaa

I did a few erroneous trials and thought it's not there.
That's an important comment. I will close this PR and create a new one